app/code/Magento/InventoryInStorePickupQuote/Model/ToQuoteAddress.php

```diff
@@ -9,13 +9,14 @@
 
 use Magento\Framework\Api\DataObjectHelper;
 use Magento\Framework\DataObject\Copy;
-use Magento\InventoryInStorePickupApi\Api\Data\PickupLocationInterface;
 use Magento\Quote\Api\Data\AddressInterface;
 use Magento\Quote\Model\Quote\Address;
 use Magento\Quote\Model\Quote\AddressFactory;
 
 /**
  * Convert Pickup Location and provided Shipping Address to Pickup Location Quote Address.
+ *
+ * @api
```

Comment:
We don't want to expose any API for InventoryInStorePickupQuote.

Reply:
ok